api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/ResumoMensalQueries.java

```diff
@@ -63,24 +69,40 @@ public static String getPatientsTransferredFromAnotherHealthFacilityDuringTheCur
             + "         ON type.encounter_id = e.encounter_id "
             + "WHERE  p.voided = 0 "
             + "       AND e.voided = 0 "
-            + "       AND e.encounter_type = %d "
+            + "       AND e.encounter_type = ${masterCard} "
```

Comment:
What does this phrase means in the specs? **Select all patients with the earliest ART Start Date from the following criterias** Do you need to select all those dates per patient, then later take the first one(one that came first)/

Reply:
I think it is, as implemented in `InitialArtStartDateCalculation`.

Reply:
in that class tarnslated to sql would be **UNION of all those subqueries(by getting patient_id and dateRequired, duplicates will be allowed here), then later have an outer query that  select patientId with MIN(dateRequired), here unique rows will be picked, then fhave an outer query that  pick a set of patientIDs, That is how I would translate that class to sql**